Xamarin.PropertyEditing/Xamarin.PropertyEditing.csproj

```diff
@@ -142,8 +141,6 @@
       <LastGenOutput>LocalizationResources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Themes\" />
-  </ItemGroup>
+  <ItemGroup />
```

Comment:
Not sure removing those folders was a conscious decision, but that empty `ItemGroup` can safely be removed.